daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-mysql.md

```diff
@@ -0,0 +1,88 @@
+---
+type: docs
+title: "MySQL"
+linkTitle: "MySQL"
+description: Detailed information on the MySQL state store component
+---
+
+## Create a MySQL Store
+
+Dapr can use any MySQL instance. If you already have a running instance of MySQL, move on to the [Create a Dapr component](#create-a-dapr-component) section.
```

Comment:
Should a tabbed view be here for both Self-Hosted and Kubernetes similar to other state store components?

Reply:
I copied from Postgre. I will look at others.

Reply:
I will use Redis as my example going forward.